draft-jhoyla-tls-key-injection.md

```diff
@@ -83,44 +81,18 @@ extra context implied.
              |
              +-----> Derive-Secret(., "ext binder"
              |                      | "res binder"
-             |                      | "imp ext binder"
-             |                      | "imp res binder", "")
```

Comment:
To make this general we need to include the "imp res binder". In particular it's necessary if we want to use this structure for ESNI. ESNI can be used with either an external PSK or a resumption PSK, and we need to be able to distinguish those cases.